lib/screens/routes/route.ex

```diff
@@ -123,6 +123,7 @@ defmodule Screens.Routes.Route do
   @spec icon(t()) :: icon()
   def icon(%{id: "Blue"}), do: :blue
   def icon(%{id: "Boat-" <> _}), do: :ferry
+  def icon(%{id: "CapeFlyer"}), do: :capeflyer
```

Comment:
I think for this fix we should just use `:cr` here to be minimally disruptive. As I mentioned in the other PR, this also applies to `FreeText` which is used in some places on DUPs, and that component will not understand this new "icon type" even if departure rows do.